build.gradle

```diff
@@ -45,7 +44,7 @@ buildscript {
   ]
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.0.1'
+    classpath 'com.android.tools.build:gradle:3.3.0-alpha12'
```

Comment:
Do we need this? Why not 3.2.0?

Reply:
Don't feel strongly, I just do what AS tells me to

Reply:
@edenman @JakeWharton To fix the warning, it is currently fixed in `3.3.0-alpha13`.